### PR TITLE
setup

### PR DESCRIPTION
**Pull request by oxica**
Первоначальные настройки

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -13,8 +19,15 @@
   </head>
   <body>
     <!-- Так добавляются фрагменты в главные HTML-файлы страниц. -->
-    <include src="./partials/example.html"></include>
-
+    <include src="./partials/heder.html"></include>
+    <include src="./partials/hero.html"></include>
+    <include src="./partials/feauters.html"></include>
```
Comment (AndriyKryvopysha) [inline src/index.html]: features wrong name of file (topic) `feauters.html`

File: src/sass/main.scss [hunk 2]
```diff
@@ -4,4 +4,17 @@
...
 // Стили для файла с примером HTML-фрагмента
-@import './example';
+@import './utils/variables';
+@import './base/reset';
+
+@import './utils/hidden';
+
+@import './heder';
+@import './hero';
+@import './feauters';
```
Comment (AndriyKryvopysha) [inline src/sass/main.scss]: features wrong name `./feauters`
header wrong name `./heder`

**Review by AndriyKryvopysha — CHANGES_REQUESTED** [review]
wrong name of topic and scss file name

**Review by AndriyKryvopysha — APPROVED** [review]

**Review by AndriyKryvopysha — APPROVED** [review]

File: src/sass/base/_reset.scss [hunk 3]
```diff
@@ -0,0 +1,25 @@
+.list {
```
Comment (OlenkaVoitsekhivska) [inline src/sass/base/_reset.scss:11]: Додати

html {
  box-sizing: border-box;
}

*,
*::before,
*::after {
  box-sizing: inherit;
}

**Pull request merged**